packages/hooks/src/useHasWallet.ts

```diff
@@ -0,0 +1,9 @@
+import useWalletsList from './useWalletsList';
+
+const useHasWallet = () => {
```

Comment:
@aum-deriv TEST + TSDoc 👀🙏🏻

Reply:
@farzin-deriv will be writing the TSDoc, tests are covered in @heorhi-deriv's [PR](https://github.com/binary-com/deriv-app/pull/9552) 🙏